app/view_models/forgot_password_show.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordShow
+  def initialize(params:, session:)
```

Comment:
Updated at 5c73e6cd

I am now passing the `param[:resend]` value instead of all params. I kept the `session` stuff as-is, because I liked having the deletion part happen in a method rather than in the controller. Easier to test that way. 🍇